mysql/resource_ti_config_defaults.go

```diff
@@ -0,0 +1,348 @@
+package mysql
```

Comment:
Where is this file coming from? Could you document please, how to regenerate it?

Reply:
Sure :)

Reply:
Thanks!